### PR TITLE
deps: update to latest chrome-devtools-frontend

### PR DESCRIPTION
**Pull request by connorjclark**
last update was 2 months ago

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6033,7 +6033,6 @@ query-string@^4.1.0:
 
 quibble@^0.6.7, quibble@connorjclark/quibble#fork:
   version "0.6.12"
-  uid f49c6c6122f08a4872a1fafc1d2aca68a7fdc2df
```
Comment (connorjclark) [inline yarn.lock:6036]: shrug, whatever

**Review by adamraine — APPROVED** [review]

File: core/lib/deprecations-strings.js [hunk 2]
```diff
@@ -127,34 +135,87 @@ const UIStrings = {
...
+   * currently specified behavior, which would throw an exception once the
+   * deprecated removal abort is no longer supported.
+   * See https://github.com/w3c/media-source/issues/19 for the discussion that
+   * led to the specification change.
    */
   mediaSourceAbortRemove:
       'Using `SourceBuffer.abort()` to abort `remove()`\'s asynchronous range removal is deprecated due to specification change. Support will be removed in the future. You should listen to the `updateend` event instead. `abort()` is intended to only abort an asynchronous media append or reset parser state.',
   /**
-   * @description TODO(crbug.com/1320346): Description needed for translation
+   * @description This is a deprecation warning to developers that occurs when
```
Comment (paulirish) [inline core/lib/deprecations-strings.js:180]: lol

Comment (connorjclark) [reply core/lib/deprecations-strings.js:180]: poor translators

Comment (adamraine) [reply core/lib/deprecations-strings.js:180]: Leaving us on a cliff-hangar

**Pull request merged**